editor/src/components/canvas/controls/select-mode/cursor-overlay.tsx

```diff
@@ -18,56 +18,30 @@ export function getCursorForOverlay(editorState: EditorState): CSSCursor | null
 
 export const CursorOverlay = React.memo(() => {
```

Comment:
I think this should be renamed, since it's not an overlay anymore :)

Reply:
fixed